dotcom-rendering/src/lib/useIsSignedIn.ts

```diff
@@ -0,0 +1,35 @@
+import { getCookie } from '@guardian/libs';
+import { useEffect, useState } from 'react';
+
+async function doOktaMethodForSignIn() {
+	const identityModule = await import('./identity');
+	return identityModule.isSignedInWithOkta();
+}
+
+function doCookieMethodForSignIn() {
```

Comment:
Similar to https://github.com/guardian/dotcom-rendering/pull/7643/files#r1228379023, I think we could rename this method so that it doesn't sound like it's _doing_ the sign in flow